src/target/riscv/riscv.h

```diff
@@ -9,8 +9,9 @@ struct riscv_program;
 #include "opcodes.h"
```

Comment:
Could you please change the include guards on lines 3-4 si that they match the convention used in the other files:

```
// current:
#ifndef RISCV_H
#define RISCV_H

// proposed:
#ifndef OPENOCD_TARGET_RISCV_RISCV_H
#define OPENOCD_TARGET_RISCV_RISCV_H
```

Please, could you check also the other existing header files - I have not commented on all of them.

Reply:
I believe it's better to address separately.
Filed an issue: https://github.com/riscv-collab/riscv-openocd/issues/1097